docs/static/setup.sh

```diff
@@ -37,7 +37,7 @@ echo ""
 echo "Keyboard Shield Selection:"
 
 prompt="Pick an keyboard:"
-options=("Kyria" "Lily58" "Corne")
+options=("Kyria" "Lily58" "Corne" "splitreus62")
```

Comment:
Please add this in the other PR for this shield, thanks!

Reply:
Will do.